src/WebJobs.Script/DependencyInjection/ScriptStartupTypeLocator.cs

```diff
@@ -335,14 +337,10 @@ void CollectError(Type extensionType, Version minimumVersion, ExtensionStartupTy
             }
         }
 
-        private bool IsDotnetIsolatedApp(IList<RpcWorkerConfig> workerConfigs)
+        private bool IsDotnetIsolatedApp(IEnumerable<FunctionMetadata> functions, IEnvironment environment)
         {
-            if (workerConfigs != null)
-            {
-                return workerConfigs.Where(config => config.Description is not null && config.Description.Language == RpcWorkerConstants.DotNetIsolatedLanguageWorkerName).Any();
```

Comment:
So were we previously checking whether there was any dotnet-isolated worker config? and if so, assuming it was dotnet-isolated? 

It seems like this would have broken once we introduced the placeholder worker, which meant this config was now always present. 

Any idea why this started showing up as an issue now?

Reply:
This would have been an issue since it was introduced in 4.26.0. I think we just detected this late.